go3/settings.py

```diff
@@ -190,7 +190,8 @@
 LOGOUT_REDIRECT_URL = '/accounts/login'
 
 # Configure Django-q message broker
-set_start_method("fork")
+# comment this out if running python on windows - fork is not a valid windows multiprocessing context. fork is declared as a start method if running django python on mac (https://code.djangoproject.com/ticket/31169)
+# set_start_method("fork")
```

Comment:
As it turns out, this is legacy and can be completely removed - if you don't do it here, I'll take it out later. It was needed on the mac for an earlier version of qcluster, now fixed.

Reply:
removed